testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/OAuthClient.java

```diff
@@ -58,6 +58,8 @@
 import org.keycloak.models.utils.KeycloakModelUtils;
 import org.keycloak.protocol.oidc.OIDCLoginProtocol;
 import org.keycloak.protocol.oidc.OIDCLoginProtocolService;
+import org.keycloak.protocol.oidc.grants.PreAuthorizedCodeGrantType;
```

Comment:
It seems that it is not used.

Reply:
Fixed in [42f5903](https://github.com/keycloak/keycloak/pull/27931/commits/42f5903f054b931386c5c249d4f2990c99ed4673)